pyteal/compiler/compiler_test.py

```diff
@@ -134,7 +134,7 @@ def test_compile_version_invalid():
         compileTeal(expr, Mode.Signature, version=1)  # too small
 
     with pytest.raises(TealInputError):
-        compileTeal(expr, Mode.Signature, version=6)  # too large
+        compileTeal(expr, Mode.Signature, version=MAX_TEAL_VERSION + 1)  # too large
```

Comment:
nit: can we just say `version=7` instead of `MAX_TEAL_VERSION +_ 1`.

Yes, it will fail again when we up the version again, but that will remind us that we have this test in place.